Formula/dish.rb

```diff
@@ -0,0 +1,23 @@
+class Dish < Formula
+  desc "Tool for parallel sysadmin of multiple hosts"
+  homepage "https://freeshell.de/~drimiks/gnu/dish.shtml"
+  url "https://freeshell.de/~drimiks/gnu/download.cgi/progs/dish/dish-1.19.1.tar.gz"
+  sha256 "eaddd3f97240f00714879cfa6e279a652bc2afad106ff1da6c5083ff27af3bf0"
+
+  depends_on "coreutils"
+  depends_on "openssl"
+
+  def install
+    mkdir_p bin
+    mkdir_p man1
+
+    system "./install.sh", "-i",
+                           "-b", bin,
+                           "-m", man1,
+                           "-q"
+  end
+
+  test do
+    system "#{bin}/dish", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.